README.md

```diff
@@ -11,38 +11,40 @@ react-router@>=1.0.0-beta3 routes like so:
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Relay from 'react-relay';
-
-import RelayNestedRoutes from 'relay-nested-routes';
-var NestedRootContainer = RelayNestedRoutes(React, Relay);
+import reactRouterRelay from 'react-router-relay';
 
 /* ... */
 
 ReactDOM.render((
-  <Router history={new BrowserHistory()}>
-    <Route component={NestedRootContainer}>
-      <Route component={App} queries={AppQueries}>
-        <Route path="/" component={Dashboard} queries={DashboardQueries}/>
-      </Route>
+  <Router history={new BrowserHistory()} createElement={reactRouterRelay()}>
+    <Route component={App} queries={AppQueries}>
+      <Route path="/" component={Dashboard} queries={DashboardQueries}/>
```

Comment:
Another option here might be to define e.g. `RelayRouter.Route`, which just extends `Router.Route` but adds `queries: React.PropTypes.obj.isRequired` to `propTypes`.